tasks/interactive-bots.md

```diff
@@ -54,19 +54,17 @@ Follow the tutorial below to test run the followup bot on your Zulip instance
 
   ```
   [api]
-  key=<api-key>
-  email=<email>
+  key=<api-key-of-the-bot>
```

Comment:
I don't agree with this change - it's intentional to allow using the bot's or an existing user's credentials for this task.